services/pub/pub-likes.service.js

```diff
@@ -0,0 +1,56 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const documentation = `<p>A measure of how many developers have liked a package. This provides a raw measure of the overall sentiment of a package from peer developers.</p>`
+
+const keywords = ['dart', 'flutter']
+
+const schema = Joi.object({
+  likeCount: Joi.number().min(0).required(),
```

Comment:
I think both this and the valid case service test can use `nonNegativeInteger`?
https://github.com/badges/shields/blob/f219e3cd193426807a5145d4992ebccebc40ec6a/services/validators.js#L11

Reply:
I have added the nonNegativeInteger for like counts.